phpstan.neon

```diff
@@ -34,37 +34,46 @@ parameters:
         -
             message: '#Access to an undefined property PhpParser\\Node::\$name\.#'
             path: %currentWorkingDirectory%/packages/framework/src/Component/Translation/ConstraintViolationExtractor.php
-        # shopsys/project-base - don't forget to add these rules to phpstan.neon in project-base (only exceptions for level 1)
+        # shopsys/project-base - don't forget to add these rules to phpstan.neon in project-base
+        # Ignore annotations in generated code
         -
             message: '#(PHPDoc tag @(param|return) has invalid value (.|\n)+ expected TOKEN_IDENTIFIER at offset \d+)#'
```

Comment:
Shouldn't we always end the message pattern with `\.` (if not `$`) for clarity and consistence? Otherwise, the patterns will be too loose.

```suggestion
            message: '#(PHPDoc tag @(param|return) has invalid value (.|\n)+ expected TOKEN_IDENTIFIER at offset \d+\.)#'
```